js/gameFunctionality.js

```diff
@@ -1,219 +0,0 @@
-// Starts the game and checks for game over
-function gameStatus() {
-  if (player.y < height / 2) {
-    gameStarted = true;
-  }
-
-  if (gameStarted) {
-    gameSpeed = 1.5;
-  }
-
-  if (gameOver) {
-    black = 0; // Using these for color-effect on both start and gameover screen, thus the reset
-    white = 255;
-    gameover.play();
-    bgmusic.stop();
-  }
-}
-
-function drawScore() {
-  push();
-  strokeWeight(1);
-  fill(200, 200, 0, colorChange / 2 + 50);
-  circle(42, 42, 40);
-  textAlign(CENTER);
-  textFont(font);
-  fill(0);
-  textSize(18);
-  text(`${score}`, 42, 39);
-  pop();
-}
-
-function drawVolumeSlider() {
-  slider = createSlider(0, 0.6, 0.3, 0.02);
-  slider.position(50, 40);
-  slider.style("width", "80px");
-}
-
-function startScreen() {
-  push();
-  textFont(font);
-  background(0);
-  fill(white);
-  circle(width / 2, height / 2, circleD);
-  fill(255);
-  textAlign(CENTER);
-
-  if (startScreenCounter > 10) {
-    // startScreenCounter starts when "orb" appeared
-    textSize(75);
-    fill(black, alphaCount * 2); // using alphaCount to have a fading in
-    text("orb", width / 2, height / 2 + 25);
-  }
-
-  if (startScreenCounter > 50) {
-    textSize(40);
-    fill(white);
-    text("Press 'orb' to play", width / 2, height / 2 + 300);
-  }
-  pop();
-  // Growing/shrinking cycle
-  if (circleD > 200) {
```

Comment:
instead of a comment you could make it into a function. then the name / code would speak for itself :)